scala-core-collections/src/main/scala/com/baeldung/scala/Arrays.scala

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.scala
```

Comment:
Let's put this in package com.baeldung.scala.collections -- there are already a few files there